app/views/data_exports/_data_export.html.erb

```diff
@@ -0,0 +1,86 @@
+<tr
+  id="<%= dom_id(data_export) %>"
+  class="text-sm font-normal text-slate-500 dark:text-slate-400"
+>
+  <td class="px-6 py-3">
+    <%= link_to data_export.id,
+    data_export_path(data_export),
+    data: {
+      turbo: false
+    },
+    class: "text-slate-900 dark:text-slate-100 font-semibold hover:underline" %>
+  </td>
+  <td class="px-6 py-3">
+    <% unless data_export.name.nil? %>
+      <%= link_to data_export.name,
+      data_export_path(data_export),
+      data: {
+        turbo: false
+      },
+      class: "text-slate-900 dark:text-slate-100 font-semibold hover:underline" %>
+    <% end %>
+  </td>
+  <td class="px-6 py-3">
+    <%= data_export.export_type.capitalize %>
+  </td>
+  <td class="px-6 py-3">
+    <% if data_export.status == 'ready' %>
+      <%= viral_pill(text: data_export.status.capitalize, color: :green) %>
+    <% else %>
+      <%= viral_pill(text: data_export.status.capitalize, color: :gray) %>
+    <% end %>
+  </td>
+  <td class="px-6 py-3">
+    <%= l(data_export.created_at.localtime, format: :full_date) %>
+  </td>
+  <td class="px-6 py-3">
+    <% unless data_export.expires_at.nil? %>
+      <%= l(data_export.expires_at.localtime, format: :full_date) %>
+    <% end %>
+  </td>
+  <td class="px-6 py-3 text-right">
```

Comment:
This comment applies to anywhere we have > 1 actions for a record. I think we should add a bit of spacing between the links. Maybe add a vertical separator.

Reply:
Fixed in fa6da0e3f3132dff1f660a2258c9b901061be91e